diff --git a/dist/commonjs/index.js b/dist/commonjs/index.js
--- a/dist/commonjs/index.js
+++ b/dist/commonjs/index.js
@@ -278,4 +278,10 @@ function expand_(str, max, maxLength, isTop) {
     }
     return acc;
 }
+module.exports = Object.assign(expand, {
+    default: expand,
+    expand,
+    EXPANSION_MAX: exports.EXPANSION_MAX,
+    EXPANSION_MAX_LENGTH: exports.EXPANSION_MAX_LENGTH,
+});
 //# sourceMappingURL=index.js.map
diff --git a/dist/esm/index.d.ts b/dist/esm/index.d.ts
--- a/dist/esm/index.d.ts
+++ b/dist/esm/index.d.ts
@@ -5,4 +5,5 @@ export type BraceExpansionOptions = {
     maxLength?: number;
 };
 export declare function expand(str: string, options?: BraceExpansionOptions): string[];
+export default expand;
 //# sourceMappingURL=index.d.ts.map
diff --git a/dist/esm/index.js b/dist/esm/index.js
--- a/dist/esm/index.js
+++ b/dist/esm/index.js
@@ -274,4 +274,5 @@ function expand_(str, max, maxLength, isTop) {
     }
     return acc;
 }
+export default expand;
 //# sourceMappingURL=index.js.map
